keps/sig-storage/20200212-volume-group.md

```diff
@@ -0,0 +1,369 @@
+Volume Group KEP
+
```

Comment:
These lines look out of place. Are they intended to be here?

Reply:
Removed